Dockerfile

```diff
@@ -4,24 +4,28 @@ ARG KUBECTL_VERSION=1.25.11
 ARG HELM_VERSION=3.12.1
 ARG KUSTOMIZE_VERSION=5.0.0
 
+
 FROM golang:1.20-bullseye as kubectl-builder
+ARG TARGETARCH
 ARG KUBECTL_VERSION
-RUN curl -sLf --retry 3 -o kubectl https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl && \
```

Comment:
In our Dockerfile, URLs must be parametrized by `TARGETARCH` to download the right binary